core/src/main/java/org/hibernate/ogm/transaction/emulated/impl/EmulatedLocalTransactionCoordinatorBuilder.java

```diff
@@ -86,7 +86,7 @@ public void commit() {
 
 		@Override
 		public void rollback() {
-			status = TransactionStatus.NOT_ACTIVE;
+			status = TransactionStatus.ROLLED_BACK;
```

Comment:
I'm not familiar with the new status codes, but are you sure this shouldn't be `MARKED_ROLLBACK` ?

Reply:
I don't think these are new, anyway I  will have a look.

Reply:
I think it's fine to use ROLLED_BACK in this scenario. There are some expectation around it and if the status is different the rollback is tried again causing event to get caught twice (and some of our tests fail).